internal/syncer/connector/confluence.go

```
@@ -0,0 +1,885 @@
+	c.download = c.downloadURL
+	c.getCurrentUser = c.currentUserName
+	return c, nil
+}
+
+// Validate validates Confluence settings and credentials.
+func (c *ConfluenceConnector) Validate(ctx context.Context) error {
+	if c == nil {
+		return fmt.Errorf("confluence connector is nil")
+	}
+	if c.wikiBase == "" {
+		return fmt.Errorf("Confluence wiki_base is required")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)